webpack/components/extensions/HostDetails/Tabs/RepositorySetsTab/RepositorySetsTab.js

```diff
@@ -96,6 +96,29 @@ EnabledIcon.propTypes = {
   isOverridden: PropTypes.bool.isRequired,
 };
 
+const OsRestrictedIcon = ({ osRestricted }) => (
+  <Tooltip
+    position="right"
+    content={<FormattedMessage
+      id="os-restricted-tooltip"
+      defaultMessage={__('OS restricted to {osRestricted}')}
```

Comment:
I'd be for a more informative message here:
```suggestion
      defaultMessage={__('OS restricted to {osRestricted}. If host OS does not match, the repo will not be available on this host.')}
```